pyproject.toml

```
@@ -1,3 +1,53 @@
+[build-system]
+requires = ["setuptools ~= 65.5", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python-algorithms"
+description = "All algorithms implemented in Python"
+version = "1.0.0"
+
+authors = [
+    { name = "Christian Clauss", email = "cclauss@me.com" },
+    { name = "Dhruv Manilawala", email = "dhruvmanila@gmail.com" },
+    { name = "John Law", email = "johnlaw.po@gmail.com" }
+]
+
+dependencies = [
+    "beautifulsoup4",
+    "fake_useragent",
+    "keras",
+    "lxml",
+    "matplotlib",
+    "numpy",
+    "opencv-python",
+    "pandas",
+    "pillow",
+    "projectq",
+    "qiskit",
+    "requests",
+    "rich",
+    "scikit-fuzzy",
+    "sklearn",
```

```suggestion
    "scikit-learn",
```
https://pypi.org/project/scikit-learn/